backend/entities/User.cs

```diff
@@ -56,14 +56,14 @@ public class User : BaseEntity
         /// </summary>
         /// <typeparam name="UserAgency"></typeparam>
         /// <returns></returns>
-        public ICollection<UserAgency> Agencies { get; } = new List<UserAgency>();
+        public ICollection<UserAgency> Agencies { get; set; } = new List<UserAgency>();
```

Comment:
The reason `set` is not included is because we don't want anyone changing the collection after the fact.